handling-null-values-in-avro-and-protobuf/kafka/README.md

```diff
@@ -0,0 +1,195 @@
+In this tutorial let's say we're tracking purchase events in Kafka with Confluent Cloud, each with an `item`, a `total_cost`, and a `customer_id`. 
+
+Let's say you're using an avro schema, and sometimes you want to set the `item` to null. How to adjust the schema to allow a null value? Let's get started. 
+
+[Sign up](https://www.confluent.io/) for a Confluent Cloud account if you haven't already. 
+
+Login, and then click 'Environments -> Create Cloud Environment' and create a cloud environment using the defaults there. 
+
+Navigate to your environment and click 'Add cluster'. Create a cluster using the default values provided. 
+
+Click 'Topics -> Add topic' to create two topics with the default values, one named 'avro-purchase' and the other 'proto-purchase' (we'll cover null values in Protobuf schemas later in the tutorial). 
+
+On the right-hand navbar, click 'API keys -> Add key -> Global access'. Download the values as you will need them to run this tutorial. 
+
+In the same navbar, click 'Clients -> Choose Your Language -> Java -> Create Schema Registry API key'. Save this key and secret as well as the URL listed in the configuration snippet. 
```

Comment:
keep the tutorial agnostic wrt where Kafka is running... suggest adding a section to the bottom like [this](https://github.com/confluentinc/tutorials/blob/master/aggregating-minmax/flinksql/README.md?plain=1#L33) though maybe just with 2 sections for now, one to run the unit test, one to run in Confluent Cloud. the third would be `Local in Docker` if you want to tackle that or that can come later